codebase2/codebase-sqlite/U/Codebase/Sqlite/Queries.hs

```diff
@@ -447,6 +449,12 @@ addSquashResultTable :: Transaction ()
 addSquashResultTable =
   executeStatements (Text.pack [hereFile|unison/sql/009-add-squash-cache-table.sql|])
 
+-- | Added as a fix because 'addSquashResultTable' was missed in the createSchema action
+-- for a portion of time.
+addSquashResultTableIfNotExists :: Transaction ()
+addSquashResultTableIfNotExists =
+  executeStatements (Text.pack [hereFile|unison/sql/010-ensure-squash-cache-table.sql|])
```

Comment:
If `OverloadedStrings` is enabled, `hereFile` is supposed to be able to produce `IsString a => a`

Reply:
I just copied all the above ones, but can swap them all if you like.

Reply:
If you want, you can just swap this one, and then that'll maybe be the one that gets copied going forward :)